stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -209,6 +218,38 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
                 GFS_Data(nb)%Coupling%skebv_wts(:,:) = skebv_wts(nb,1:GFS_Control%blksz(nb),:)
             end do
          end if
+         if (GFS_Control%do_spp) then
+            DO v=1,GFS_Control%n_var_spp
+               select case (trim(GFS_Control%spp_var_list(v)))
+               case('pbl')
+       print *,"SPP: spp_wts_pbl is ", v, maxval(spp_wts(:,:,:,v))
+                 do nb=1,Atm_block%nblks
+                     GFS_Data(nb)%Coupling%spp_wts_pbl(:,:) = spp_wts(nb,1:GFS_Control%blksz(nb),:,v)
+                     GFS_Data(nb)%Intdiag%spp_wts_pbl(:,:) = spp_wts(nb,1:GFS_Control%blksz(nb),:,v)
```

Comment:
See my comment above about the unnecessary Intdiag variables.

Reply:
Removed references to Intdiag variables.